attorch/layers.py

```diff
@@ -244,23 +245,14 @@ def initialize(self, init_noise=1e-3):
         if self.bias is not None:
             self.bias.data.fill_(0)
 
-    def sigma_l1(self):
+    def sigma_l1(self, offset=0.1):
         return (self.sigma - self.sigma_eps).abs().mean()
-        
-    def sigma_l2(self):
-        return (self.sigma - self.sigma_eps).pow(2).mean()
 
     def feature_l1(self, average=True):
         if average:
             return self.features.abs().mean()
         else:
-            return self.features.abs().sum()
-             
-    def weight_l1(self, average=True):
```

Comment:
why are you removing these? I believe I have code using these regularizations.

Reply:
must have been a mistake